### PR TITLE
Better landing page

### PR DESCRIPTION
**Pull request by jonathanmetzman**

### CODE REVIEW

**Comment by jonathanmetzman** [comment]
@olivekl What do you think of this?
I think it would be good to add some screenshots too.

File: docs/clusterfuzzlite.md [hunk 1]
```diff
@@ -7,19 +7,32 @@ permalink: /
...
-based off [ClusterFuzz]. ClusterFuzzLite is designed to run as workflows on
-[continuous integration] (CI) systems, which means it is easy to set up and
-provides a familiar interface for users.
 
-Currently ClusterFuzzLite fully supports [GitHub Actions]. Extending support to
-other CI systems is in the works.
+ClusterFuzzLite makes [fuzzing] part of [Continuous Integration (CI)].
+ClusterFuzzLite is based on [ClusterFuzz].
+
+By fuzzing your code, ClusterFuzzLite can make it more secure as fuzzing is
```
Comment (olivekl) [inline docs/clusterfuzzlite.md:14]: Small suggestion:

`Fuzzing is a highly effective technique for finding bugs in software. ClusterFuzzLite makes your code more secure by fuzzing changes before they even enter the codebase.`

**Review by olivekl — COMMENTED** [review]
One suggested change, otherwise LGTM!

**Comment by jonathanmetzman** [comment]
> One suggested change, otherwise LGTM!

Do you think my changes make the page better? I thought it was lacking in detail.

**Comment by olivekl** [comment]
Yes, I think it's an improvement. Just be aware of the potential overlap with the Overview page (having a bulleted list of features on both the Landing page and Overview page). Would it make sense to merge those lists on one page or the other?

**Comment by jonathanmetzman** [comment]
> Yes, I think it's an improvement. Just be aware of the potential overlap with the Overview page (having a bulleted list of features on both the Landing page and Overview page). Would it make sense to merge those lists on one page or the other?

Yeah, I was thinking the same. I'm going to merge that page into this one.

**Review by oliverchang — COMMENTED** [review]
LGTM

**Pull request merged**